packages/mdc-fab/mdc-fab.scss

```diff
@@ -31,6 +32,15 @@
   @include mdc-fab--mini_;
 }
 
+.mdc-fab--extended {
+  @include mdc-typography(button);
```

Comment:
I would move this include into the extended_ mixin as well, thinking forward to if we end up with a reason to expose it. (We do the same with e.g. `mdc-button-base_`)